internal/ocm/resource/cluster.go

```diff
@@ -33,14 +33,22 @@ func (c *Cluster) Build() (object *cmv1.Cluster, err error) {
 
 func (c *Cluster) CreateNodes(autoScalingEnabled bool, replicas *int64, minReplicas *int64,
 	maxReplicas *int64, computeMachineType *string, labels map[string]string,
-	availabilityZones []string, multiAZ bool) error {
+	availabilityZones []string, multiAZ bool, workerDiskSize *int64) error {
 	nodes := cmv1.NewClusterNodes()
 	if computeMachineType != nil {
 		nodes.ComputeMachineType(
 			cmv1.NewMachineType().ID(*computeMachineType),
 		)
 	}
 
+	if workerDiskSize != nil {
+		nodes.ComputeRootVolume(
+			cmv1.NewRootVolume().AWS(
+				cmv1.NewAWSVolume().Size(int(*workerDiskSize)),
```

Comment:
Do we need to set default value here or just use the default value in the OCM backend?
What is the behavior in the CLI?

Reply:
In the CLI, if the size is not set or identical to the flavor's default size, we do nothing and let the backend handle the defaults.